content/pages/sprint/index.rst

```diff
@@ -13,6 +13,8 @@ hosted by sciwork.
 Scisprint 2021
 ==============
 
+* `Scisprint 2021 Mid August: Python ZH TW Doc <{filename}2021/08mid-pycht.rst>`__
```

Comment:
The sprints are ordered reversed chronically.  Could you move it lower than the August entry?

You can rename the other one to be Late August (but keep the URL unchanged).  But I think this is optional.

Reply:
Fixed it, I've rename it to "Late August" and also updated the order.